rfcs/generic-functions.md

```diff
@@ -0,0 +1,110 @@
+# generic functions
```

Comment:
All the other RFCs have lower case titles?

Reply:
> Change _VERSION global to "Luau"

> Array-like table types

It looks like it's sentence case rather than title case, but the only examples of pure-lowercase I see are RFCs where the title is the name of a function like `table.freeze`.

Reply:
Yeah, this is correct. I will change the template to that effect; the title should say "Generic functions"